### PR TITLE
Playground button css fixed

### PR DESCRIPTION
**Pull request by ivikash**
Hey, 

The text in button's of playground didnt feel right, with this PR I intend to fix the padding issue

> Before
> screen shot 2016-08-03 at 3 21 00 pm
> 
> After
> screen shot 2016-08-03 at 3 28 31 pm

### CODE REVIEW

**Comment by dlongley** [comment]
If this kind of problem needs to be fixed, it should have some kind of bootstrap fix; I don't think we should start doing custom CSS for it.

**Comment by ivikash** [comment]
Infact at first I thought the same. Lets make it generic. but then the form inside gist started breaking. So what I will do is, I will fix those in bootstrap.css itself :)

**Comment by dlongley** [comment]
If we need to upgrade bootstrap, that's fine, but let's not make changes to bootstrap.css unless they are the types of fixes that would be submitted to that project directly. We should be able to use vanilla bootstrap properly to get a nice looking site.

**Comment by ivikash** [comment]
Updated changes
screen shot 2016-08-03 at 11 48 06 pm

File: playground/playground.css [hunk 1]
```diff
@@ -60,14 +56,15 @@
...
-}
-
 #page-playground .read-only {
   opacity: .6;
 }
 
+#page-playground .btn {
+  height: 30px;
+  padding: 0 0.7em;
+}
```
Comment (dlongley) [inline playground/playground.css]: Ideally, there wouldn't be anything custom required here at all. If the buttons still look wrong, we must not be doing something quite right with bootstrap. There default theme looks just fine (and we're trying to use that, AFAIK).

**Comment by gkellogg** [comment]
This PR is languishing, we should either commit to close.

cc/ @msporny @davidlehn @dlongley @lanthaler.

**Pull request closed without merging**